js/src/product-feed/product-statistics/create-campaign-notice.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Notice } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import AddPaidCampaignButton from '.~/components/paid-ads/add-paid-campaign-button';
+import useMCProductStatistics from '.~/hooks/useMCProductStatistics';
+import useAdsCampaigns from '.~/hooks/useAdsCampaigns';
+
+const CreateCampaignNotice = () => {
+	const { hasFinishedResolution: productsResolution, data: products } =
+		useMCProductStatistics();
+	const { loaded: campaignsLoaded, data: campaigns } = useAdsCampaigns();
+
+	const isLoading =
+		! productsResolution || products?.loading || ! campaignsLoaded;
+
+	const inValidData =
+		( productsResolution && ! products ) ||
+		( campaignsLoaded && ! campaigns );
+
+	if (
+		isLoading ||
```

Comment:
@kt-12 We have more or less the same conditions when using `isLoading` in `inValidData`. Can we simplify it please?
```suggestion
if (
		isLoading ||
		! campaigns ||
		campaigns?.length > 0 ||
		! products ||
		products?.statistics?.active === 0
	) {
```

Reply:
Done.